kura/distrib/src/main/resources/common/find_net_interfaces.py

```diff
@@ -11,57 +11,58 @@
 # Contributors:
 #  Eurotech
 #
-import subprocess
 import sys
 import logging
+import os
 
-ETHERNET_TYPES = ['ethernet', 'eth', 'wired']
-WIRELESS_TYPES = ['wifi', 'wireless']
+def get_interface_names():
+    if not os.path.exists("/sys/class/net"):
+        logging.error("'/sys/class/net' does not exists, unable to read interface names. Exiting.")
+        sys.exit(1)
+
+    return os.listdir("/sys/class/net")
 
 def get_eth_wlan_interfaces_names():
-    """Reads the network interface names using 'nmcli dev' command.
+    """Reads the network interface names present on the device.
 
     It is assumed that at least one ethernet interface is present.
 
+    Requirements:
+        "/sys/class/net" directory must exist
+
     Returns:
         tuple of lists (eth_names, wlan_names) where:
             'eth_names' are the found ethernet interface names sorted by name;
             'wlan_names' are the found wireless interface names sorted by name, might be an empty list.
     """
-    cmd_output = subprocess.check_output(['nmcli', 'dev']).decode(sys.stdout.encoding).strip()
-    # list comprehension to remove empty items
-    lines = [x.strip() for x in cmd_output.split('\n') if len(x.strip()) > 0]
-
-    # removing header
-    del lines[0]
+    ethernet_interface_names = list()
+    wireless_interface_names = list()
 
-    ethernet_inteface_names = list()
-    wireless_inteface_names = list()
+    interface_names = get_interface_names()
 
-    for line in lines:
-        row = [x.strip() for x in line.split(' ') if len(x.strip()) > 0]
-        
-        interface_name = row[0].lower()
-        interface_type = row[1].lower()
-
-        if interface_type in ETHERNET_TYPES:
-            ethernet_inteface_names.append(interface_name)
-        
-        if  interface_type in WIRELESS_TYPES:
-            wireless_inteface_names.append(interface_name)
+    # en, et = ethernet
+    # wl = wireless lan
+    # ww = wireless wan
+    # eno<index> = on-board ethernet
+    # ens<slot> = hotplug on slot <slot> ethernet
+    # enp<index> = pci ethernet
```

Comment:
One small detail: while reading this for a moment I thought you forgot to delete some commented out code. Can you add a small comment just to explain that this is your criteria for differentiating the interfaces types?

Something like:

```suggestion
    # We use the interface name to differentiate their type. Criteria used below:
    # en*, et* -> ethernet
    # wl* -> wireless lan
    # ww* -> wireless wan
    # eno<index> -> on-board ethernet
    # ens<slot> -> hotplug on slot <slot> ethernet
    # enp<index> -> pci ethernet
```

Reply:
Maybe the code is clear enough. I think we can remove the comment. What do you think?

Reply:
Good idea. The code is clear enough that it doesn't require explainatory comment 👍